relayer/src/main.rs

```diff
@@ -1,23 +1,27 @@
 use std::time::Duration;
 
 use clap::{Args, Parser, Subcommand};
-use message_relayer::{all_token_transfers, paid_token_transfers};
+use ethereum_beacon_client::BeaconClient;
+use gclient::{GearApi as GclientGearApi, WSAddress};
```

Comment:
```suggestion
use gclient::{GearApi as GClientGearApi, WSAddress};
```

More conforms to the code conventions, isn't it?